### PR TITLE
Kube-state-metrics replica count

### PR DESCRIPTION
**Pull request by jpweber**
Adds in values to set kube-state-metrics replica count to 2 to reflect what is actually running in prod.

### CODE REVIEW

**Comment by jpweber** [comment]
Should the change and subsequent PR been done on the https://github.com/astronomer/terraform-google-astronomer-cloud repo rather than this one?

**Comment by sjmiller609** [comment]
> Should the change and subsequent PR been done on the https://github.com/astronomer/terraform-google-astronomer-cloud repo rather than this one?

discussed in PM: the above linked is the old way and not preferred to the change in this PR.

File: prod/cloud/locals.tf [hunk 1]
```diff
@@ -383,5 +383,7 @@ fluentd:
       key: dynamic-pods
       operator: Equal
       value: "true"
+kube-state:
+  replicas: 2
```
Comment (sjmiller609) [inline prod/cloud/locals.tf]: so this change looks good, but it's not supported in the underlying helm chart. just need a PR at the following location first https://github.com/astronomer/astronomer/blob/816ae62c7ff08488a8da6f1aa2c677c3c1a778d2/charts/kube-state/templates/kube-state-deployment.yaml#L16

Then this PR after that. So basically first make sure the configuration is available in the helm chart, then add the configuration to our environments.

Comment (jpweber) [reply prod/cloud/locals.tf]: see https://github.com/astronomer/astronomer/pull/599

**Pull request merged**